istioctl/cmd/istioctl/rbac.go

```diff
@@ -16,14 +16,153 @@ package main
 
 import (
 	"fmt"
+	"io"
+	"strings"
 
+	"github.com/ghodss/yaml"
+	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/cobra"
 
 	rbacproto "istio.io/api/rbac/v1alpha1"
 	"istio.io/istio/mixer/adapter/rbac"
 	"istio.io/istio/pilot/pkg/model"
 )
 
+// list provides a command named list that allows user to query information under current
+// Istio RBAC policies.
+func list() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Query information under current Istio RBAC policies.",
+		Long: `
+This command lets you query information under current Istio RBAC policies.
+You can list all permissions a subject has, or all members that are allowed to access a service`,
```

Comment:
Nit: Should we use `member` or `principal` or `identity` here? `member` seems a new term that is never used in the istio related documentation, so just make sure the user understand what it is.

Reply:
I think `member` is ok.